app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <title>IDseq</title>
+    <meta name="description" content="An unbiased global software platform that helps scientists identify pathogens in metagenomic data.">
+    <meta name="keywords" content="Infectious disease sequencing, genomics, Chan Zuckerberg Initiative, CZI, Chan Zuckerberg Biohub, CZ Biohub, science, next gen sequencing, research, biology, virology, bioinformatics, computational biology, global health">
```

Comment:
Infectious disease, sequencing, metagenomic sequencing, NGS, @cdebourcy  should help edit the list